tests/configs/standard/config.yaml

```diff
@@ -0,0 +1,352 @@
+      # AUTO_REPO_URLS: "https://github.com/lsst-sqre/system-test,https://github.com/rubin-dp0/tutorial-notebooks"  # No longer needed
+      CULL_KERNEL_IDLE_TIMEOUT: "432000"  # These might be set from group?
+      CULL_KERNEL_CONNECTED: "True"
+      CULL_KERNEL_INTERVAL: "300"
+      DAF_BUTLER_REPOSITORY_INDEX: "s3://butler-us-central1-repo-locations/data-repos.yaml"
+      FIREFLY_ROUTE: /portal/app
+      HUB_ROUTE: /nb/hub
+      # JS9_ROUTE: /js9  # No longer used?
+      JUPYTERHUB_ADMIN_ACCESS: "1"  # Do we need this?!?
+      JUPYTERHUB_API_URL: http://hub.nublado2:8081/nb/hub/api
+      JUPYTERHUB_BASE_URL: /nb/
+      JUPYTERHUB_DEFAULT_URL: /lab
+      JUPYTERHUB_OAUTH_CLIENT_ALLOWED_SCOPES: '[]' # needed?
+      NO_ACTIVITY_TIMEOUT: "432000"  # Also from group?
+      NO_SUDO: "TRUE"
+      S3_ENDPOINT_URL: "https://storage.googleapis.com"
+      SODA_ROUTE: /api/image/soda
+      TAP_ROUTE: /api/tap
+      # WORKFLOW_ROUTE: /wf  # No longer used
+      # Note that other environment variables will be injected by the
+      # Lab controller (to match KubeSpawner's behavior).  These
+      # must be calculated from other Lab-spawn-time values, and
+      # include:
+      # ACCESS_TOKEN
+      # CPU_LIMIT
+      # DEBUG
+      # EXTERNAL_GID
+      # EXTERNAL_GROUPS
+      # JPY_API_TOKEN
+      # JUPYTERHUB_ACTIVITY_URL
+      # JUPYTERHUB_CLIENT_ID
+      # JUPYTERHUB_OAUTH_ACCESS_SCOPES
+      # JUPYTERHUB_OAUTH_CALLBACK_URL
+      # JUPYTERHUB_OAUTH_SCOPES  # Alias for JUPYTERHUB_OAUTH_ACCESS_SCOPES
+      # JUPYTERHUB_SERVICE_PREFIX
+      # JUPYTERHUB_SERVICE_URL
+      # JUPYTERHUB_USER
+      # JUPYTER_IMAGE
+      # JUPYTER_IMAGE_SPEC  # Alias for JUPYTER_IMAGE
+      # MEM_GUARANTEE  # in bytes
+      # MEM_LIMIT      # in bytes
+      # RESET_USER_ENV
+      # Additionally, some environment variables will be set from the
+      # secrets below.  These will be paths to injected credential
+      # files.
+      # EXTERNAL_INSTANCE_URL is a special case: it's
+      # constant-per-instance, but you can't template values files
+    form:
+      # -- Lab form restrictions.  It's a list of objects where each
+      # object has three fields, 'type', 'value', and 'groups'.  'type'
+      # must be one of "size", "image", or "tag".  'value' is a regular
+      # expression matching a string that is an allowed value for 'type'.
+      # 'groups', if specified, is a list of group names allowed to use
+      # a resource where 'type' matches 'value'.  An empty list is
+      # unrestricted.
+      restrictions: []
+      # - type: "size"
+      #   value: "large"
+      #   groups:
+      #   - "admins"
+      #   - "power_users"
+      # - type: "image"
+      #   value: "docker.io/lsstsqre/admin-*"
+      #   groups: [ "admins" ]
+      # - type "tag"
+      #   value: ".*_exp_.*"
+      #  groups:
+      #  - "experimental_users"
```

Comment:
Oh, I see what this is now. I would leave this off for the first pass; I don't think we really understand how this should work, and I don't think we want to commit to an implementation without a design session.